js_modules/dagster-ui/packages/ui-core/src/runs/LogsRowComponents.tsx

```diff
@@ -125,22 +126,20 @@ const OpColumnTooltipStyle = JSON.stringify({
   left: 1,
 });
 
-const timestampFormat = memoize((timezone: string) => {
-  return new Intl.DateTimeFormat(navigator.language, {
-    hour: '2-digit',
-    minute: '2-digit',
-    second: '2-digit',
-    hourCycle: 'h23',
-    timeZone: timezone === 'Automatic' ? browserTimezone() : timezone,
-  });
-});
-
-const fractionalSecondFormat = memoize((locale: string) => {
-  return new Intl.NumberFormat(locale, {
-    minimumFractionDigits: 3,
-    maximumFractionDigits: 3,
-  });
-});
```

Comment:
Moved this into the formatter function above so that `AM/PM` will be rendered in the correct place. I don't recall why we weren't just using `fractionalSecondDigits` in there already.